### DISCUSSION TITLE
Image upload style

### DISCUSSION THREAD

**Discussion by monkeycc** [flet-dev/flet#1063] [Q&A]
https://github.com/gradio-app/gradio
https://www.gradio.app/demos/

I want to achieve similar effects with flex

I think it can be realized with flex

But they are simple and can be used directly

We only have simple buttons
![微信截图_20230219182827](https://user-images.githubusercontent.com/6490927/219942463-6a182c40-63b5-4d6e-93b0-105c33d4d5d9.png)

![screenshot](https://user-images.githubusercontent.com/6490927/219942335-f01752db-15a1-4c9b-b6d6-77a381980a57.gif)

![微信截图_20230219182416](https://user-images.githubusercontent.com/6490927/219942247-457912d0-a467-4d32-ab43-9d187dee5683.png)

**Comment by FeodorFitsner** [comment]
FilePicker is non-visual control - you can open it on any event in your app, on a click of any button.
